packages/integration/src/lib/search/search-results-tool/search-results-tool.component.ts

```diff
@@ -525,11 +525,15 @@ export class SearchResultsToolComponent implements OnInit, OnDestroy {
     }
   }
 
-  toggleTopPanel() {
-    if (this.topPanelState === 'expanded') {
-      this.topPanelState = 'collapsed';
+  toggleTopPanel(e: MouseEvent = null) {
+    if (e !== null && (e.target as any).className !== 'igo-panel-title') {
```

Comment:
to simplify as much as possible, i think the event shouldn't be initialized (undefined) and then it will only need to be:

if (!e && (e.target as any)[...] {
[...]